# HG changeset patch
# User Jeff Walden <jwalden@mit.edu>
# Date 1365111823 25200
# Node ID 489ab986ea69e1ba7746e383a267483db9d861ea
# Parent  04f2c0f0a220b409af15722eaec46304979f04ac
#include jsscriptinlines.h in SPSProfiler.cpp so that it builds in no-jit configurations.  No bug, r=sparky

diff --git a/js/src/vm/SPSProfiler.cpp b/js/src/vm/SPSProfiler.cpp
--- a/js/src/vm/SPSProfiler.cpp
+++ b/js/src/vm/SPSProfiler.cpp
@@ -11,16 +11,18 @@
 #include "jsscript.h"
 
 #include "methodjit/MethodJIT.h"
 #include "methodjit/Compiler.h"
 
 #include "vm/SPSProfiler.h"
 #include "vm/StringBuffer.h"
 
+#include "jsscriptinlines.h"
+
 using namespace js;
 
 using mozilla::DebugOnly;
 
 SPSProfiler::SPSProfiler(JSRuntime *rt)
   : rt(rt),
     stack_(NULL),
     size_(NULL),
